kbase-extension/static/kbase/js/common/jobCommChannel.js

```diff
@@ -379,32 +395,42 @@ define([
                     break;
 
                 // job information for one or more jobs
-                // Object with keys jobId and values { jobId: jobId, jobInfo: { ...job params... } }
+                // Object with keys jobId and values { jobInfo: { ...job params... } }
                 case BACKEND_RESPONSES.INFO:
                     Object.keys(msgData).forEach((_jobId) => {
-                        this.sendBusMessage(JOB, msgData[_jobId].job_id, RESPONSES.INFO, {
-                            jobId: msgData[_jobId].job_id,
-                            jobInfo: msgData[_jobId],
-                        });
+                        const jobData = msgData[_jobId];
+                        if (Jobs.isValidJobInfoObject(jobData)) {
+                            this.sendBusMessage(JOB, _jobId, RESPONSES.INFO, {
+                                jobInfo: jobData,
+                            });
+                        } else {
+                            this.reportCommMessageError({ msgType, msgData: jobData });
+                        }
                     });
                     break;
 
                 case BACKEND_RESPONSES.LOGS:
-                    this.sendBusMessage(JOB, msgData.job_id, RESPONSES.LOGS, {
-                        jobId: msgData.job_id,
-                        logs: msgData,
-                        latest: msgData.latest,
+                    Object.keys(msgData).forEach((_jobId) => {
+                        const jobData = msgData[_jobId];
```

Comment:
this data structure has changed slightly -- the logs endpoint now returns data in the same format as the other endpoints, i.e.
```py
{
    job_id_1: job_1_data,
    job_id_2: job_2_data,
    ...
}
```